predicators/approaches/bridge_policy_approach.py

```diff
@@ -365,10 +366,79 @@ def __init__(self,
+            params_space=Box(0, len(train_tasks), (1, )),
+        )
+        initial_options.add(self.CallPlanner)
+        self._initial_options = initial_options
         self.mapleq=MapleQApproach(self._get_current_predicates(), \
                                    self._initial_options, self._types, \
                                     self._action_space, self._train_tasks)
+        self._current_control = ""
+        option_policy = self._get_option_policy_by_planning(
+            self._train_tasks[0], CFG.timeout)
+        self._current_policy = utils.option_policy_to_policy(
+            option_policy,
+            max_option_steps=CFG.max_num_steps_option_rollout,
+            raise_error_on_repeated_state=True,
+        )
+        self._bridge_called_state = State(data={})
+        self._policy_logs: List[str] = []
+
+    def _Can_plan(self, state: State, _: Sequence[Object]) -> bool:
+        if (self.mapleq._q_function._vectorize_state(state) !=  # pylint: disable=protected-access
+                self.mapleq._q_function._vectorize_state(  # pylint: disable=protected-access
+                    self._bridge_called_state)).any():  # pylint: disable=protected-access
+            return True
+        return False
+
+    def call_planner_policy(self, state: State, _: Dict, __: Sequence[Object],
+                            params: Array) -> Action:
+        """policy for CallPlanner option."""
+        self._current_control = "planner"
+        # create a new task where the init state is our current state
+        current_task = Task(state, self._train_tasks[int(params[0])].goal)
+        option_policy = self._get_option_policy_by_planning(
+            current_task, CFG.timeout)
+        self._current_policy = utils.option_policy_to_policy(
+            option_policy,
+            max_option_steps=CFG.max_num_steps_option_rollout,
+            raise_error_on_repeated_state=True,
+        )
+
+        return Action(np.array([0.0, 0.0, 0.0], dtype=np.float32))
+
+    def call_planner_nsrt(self) -> NSRT:
+        """CallPlanner NSRT."""
+        parameters: Sequence[Variable] = []
+        option_vars = parameters
+        option = self.CallPlanner
+        preconditions = {LiftedAtom(self.CanPlan, [])}
+        add_effects: Set[LiftedAtom] = set()
+        delete_effects: Set[LiftedAtom] = set()
+
+        ignore_effects: Set[Predicate] = set()
+        call_planner_nsrt = NSRT("CallPlanner", parameters, preconditions,
+                                 add_effects, delete_effects, ignore_effects,
+                                 option, option_vars, self.planner_sampler)
+        return call_planner_nsrt
+
+    def planner_sampler(self, state: State, goal: Set[GroundAtom],
+                        rng: np.random.Generator,
+                        objs: Sequence[Object]) -> Array:
+        """sampler for CallPlanner Option."""
+        del state, goal, objs  # unused
+        # Note: just return a random value from -1 to 1
```

Comment:
could you just make the `call_planner_nsrt` not take in any continuous params, and then use a `NullSampler` (which I think we've implemented somewhere in the codebase?)